CHANGELOG.md

```diff
@@ -50,7 +50,7 @@
 * ARROW-7631 - [C++][Gandiva] return zero if there is an overflow while converting a decimal to a lower precision/scale
 * ARROW-7680 - [C++][Dataset] Partition discovery is not working with windows path
 * ARROW-7701 - [C++] [CI] Flight test error on macOS
-* ARROW-7713 - [Java] TastLeak was put at the wrong location
+* ARROW-7713 - [Java] TasteLeak was put at the wrong location
```

Comment:
Please revert changes in this file, it is generated from JIRA, so if you want to fix the typos, you need to fix the JIRA issue titles